Sources/MlemMiddleware/FeedLoaders/Generics/Fetcher.swift

```diff
@@ -29,7 +29,7 @@ enum LoadingResponse<Item: FeedLoadable> {
 }
 
 public class Fetcher<Item: FeedLoadable> {
```

Comment:
Any chance fetchers be made `@Observable` while we're here? Technically OOS but the sort mode icon in the feed occasionally doesn't update because the sort mode status is stored in `PostFetcher`, which isn't `@Observable`.